.changeset/slimy-pumpkins-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+wrangler: major
```

Comment:
Minor version bump to kv-asset-handler, workers-shared, workflows-shared
Major bump to miniflare

Reply:
8d13f427a